applications/ConstitutiveLawsApplication/custom_constitutive/composites/serial_parallel_rule_of_mixtures_law.cpp

```diff
@@ -254,40 +262,39 @@ void SerialParallelRuleOfMixturesLaw::IntegrateStrainSerialParallelBehaviour(
     const ConstitutiveLaw::StressMeasure& rStressMeasure
 )
 {
-    const std::size_t voigt_size = this->GetStrainSize();
-    const int num_parallel_components = inner_prod(mParallelDirections, mParallelDirections);
-    const int num_serial_components = voigt_size - num_parallel_components;
+    const SizeType voigt_size = GetStrainSize();
```

Comment:
Question. Do we do not assume the Voigt size from the Dimension size?, sorry I forgot that detail

Reply:
yes, the `GetStrainSize()` return a static `SizeType`, I can remove it if you think is best to access it directly.

Reply:
I those method were override somewhere else it could make some sense, but here I thinbk it is better to just use it.